File: MedAPI-UI/src/app/models/labUploadResult.ts
import { MedicUser } from './medicuser.model';

export class LabUploadResult  {
  
    id: number  = 0;
    file: File;
    comments: string = '';
    fileName: string = '';
    user_id: number = 0;
    patient_docNumber : number = 0;
    uploadedBy: string = '';
    dateUploaded: Date = new Date();
    labId: number = 0;
    medicId: number = 0;
    
    
  }

File: MedAPI-Core/Data/Model/LabUploadResult.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace Data.Model
{
    public partial class LabUploadResult
    {
        public long Id { get; set; }
        public long UserId { get; set; }
        public long? LabId { get; set; }
        public long? MedicUserId { get; set; }
        public string FileName { get; set; }
        public string Comments { get; set; }
        public byte[] FileContent { get; set; }
        public DateTime? DateUploaded { get; set; }

        public virtual Lab Lab { get; set; }
        public virtual Medic MedicUser { get; set; }
        public virtual User User { get; set; }
    }
}
